include/swift/SIL/AbstractionPattern.h

```diff
@@ -527,7 +601,11 @@ class AbstractionPattern {
   /// Return a pattern corresponding to the 'self' parameter of the
   /// current C function imported as a method.
   AbstractionPattern getCFunctionAsMethodSelfPattern(CanType paramType) const;
-  
+
+  /// Return a pattern corresponding to the 'self' parameter of the
+  /// current C++ method.
+  AbstractionPattern getCXXMethodSelfPattern(CanType paramType) const;
+
```

Comment:
These were grouped by the kinds they applied to, so could you put this below `getCFunctionAsMethodFormalParamPattern`?

Did you not need a `getCXXMethodFormalParamPattern`?

Reply:
Turns out the other ones were dead code.

Reply:
All the better, thanks!